presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -470,6 +471,12 @@ public enum LeftJoinArrayContainsToInnerJoinStrategy
         ALWAYS_ENABLED
     }
 
+    public enum HistoryBasedOptimizerInputStatisticsCheckStrategy
```

Comment:
Will add one more option "HISTORY" in the next PR, which is to decide whether to skip metadata access based on historical runs, i.e. if input statistics change much in the historical runs